data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/util/KafkaSourceSecurityConfigurer.java

```diff
@@ -269,6 +283,15 @@ public static void setAuthProperties(Properties properties, final KafkaSourceCon
                 properties.put("ssl.engine.factory.class", InsecureSslEngineFactory.class);
             }
         }
+        if (sourceConfig.getSchemaConfig().getType() == SchemaRegistryType.AWS_GLUE) {
+            Map<String, Object> configs = new HashMap();
+            configs.put(AWSSchemaRegistryConstants.AWS_REGION, sourceConfig.getAwsConfig().getRegion());
+            configs.put(AWSSchemaRegistryConstants.AVRO_RECORD_TYPE, AvroRecordType.GENERIC_RECORD.getName());
+            configs.put(AWSSchemaRegistryConstants.CACHE_TIME_TO_LIVE_MILLIS, "86400000");
+            configs.put(AWSSchemaRegistryConstants.CACHE_SIZE, "10");
+            configs.put(AWSSchemaRegistryConstants.COMPATIBILITY_SETTING, Compatibility.FULL);
```

Comment:
This may require input from customer, but we can leave it like this for now.

Reply:
TTL is set to 1day. cache size default is 200, I changed it to 10, based on the example code you provided. I am to not change to 10 and keep it to the default value.